test/test_configuration.rb

```diff
@@ -12,7 +12,45 @@ def test_change_in_configuration_keep_default_untouched
   end
 
   def test_options
+    hundle_options
     assert @config.options
     assert_includes @config.options, :args
   end
+
+  def test_option_templates
+    hundle_options('--templates')
+    assert(@config.options)
+    assert_includes(@config.options, :templates)
+  end
+
+  def test_option_version
+    hundle_options('-v')
+    assert(@config.options)
+    assert_includes(@config.options, :version)
+  end
+
+  def test_invalid_option
+    assert_throws_and_match(Gem2Rpm::Configuration::ConfigurationError,
```

Comment:
I wanted to check both the type of the exception object's class, and the message (e.to_s) of the exception object.
Because assert_raises looks only check the type of the exception class.